home/wagtail_hooks.py

```diff
@@ -211,7 +211,11 @@ def page(self, obj):
     def time(self, obj):
         if obj.pages:
             return [
-                (f"{p.value['time']}" if p.value and "time" in p.value else "")
+                (
+                    f"{p.value['time']}"
+                    if p.value and "time" in p.value and p.value["time"]
```

Comment:
Is `p.value` here just a dictionary? Could we do `f"{p.value["time"]}" if p.value and p.value.get("time") else ""`?

Or maybe we can split this out into a method, since this is repeated, and rather do the ask for forgiveness pattern?
```
def get_field_value(page: Page, field: str) -> str:
    try:
        if value := p.value[field]:
            return f"{value}"
        else:
            return ""
    except AttributeError, TypeError:
        return ""
```